pkg/vulnerability/vulnerability.go

```diff
@@ -118,24 +126,14 @@ func (c Client) Filter(ctx context.Context, vulns []types.DetectedVulnerability,
 			return nil, xerrors.Errorf("failed to apply the policy: %w", err)
 		}
 	}
-
 	sort.Slice(vulnerabilities, func(i, j int) bool {
-		if vulnerabilities[i].PkgName != vulnerabilities[j].PkgName {
-			return vulnerabilities[i].PkgName < vulnerabilities[j].PkgName
-		}
-		ret := dbTypes.CompareSeverityString(
-			vulnerabilities[j].Severity, vulnerabilities[i].Severity,
-		)
-		if ret != 0 {
-			return ret > 0
-		}
-		return vulnerabilities[i].VulnerabilityID < vulnerabilities[j].VulnerabilityID
+		return vulnerabilities[i].Less(vulnerabilities[j])
 	})
 	return vulnerabilities, nil
 }
 
 func applyPolicy(ctx context.Context, vulns []types.DetectedVulnerability, policyFile string) ([]types.DetectedVulnerability, error) {
-	policy, err := ioutil.ReadFile(policyFile)
+	policy, err := ioutil.ReadFile(filepath.Clean(policyFile))
```

Comment:
Do we have to add `filepath.Clean` to all file-related functions? I'm feeling like it is meaningless.

Reply:
@knqyf263 Read this
https://stackoverflow.com/questions/52320708/how-to-handle-gosec-linter-warning-potential-file-inclusion-via-variable

Reply:
Yes, but in some cases, a path is hard-coded. It doesn't make sense to apply `filepath.Clean`.

Reply:
So if I don't clean it I will get linter errors.

Reply:
You can disable the linter. We don't have to follow all warnings from linters.

Reply:
@knqyf263 Done

Reply:
Can we disable only G304 as follows?
https://github.com/securego/gosec#selecting-rules

Reply:
@knqyf263 Already removed
https://github.com/aquasecurity/trivy/pull/679/files#diff-9917ddc9f1c3304218f7269265b746d997c5c0615478177b5fceecd33ef47cb5R54